pkgs/applications/window-managers/sway/default.nix

```
@@ -8,8 +8,18 @@
 , isNixOS ? false
 
 , enableXWayland ? true
+, systemdSupport ? stdenv.isLinux
+, dbusSupport ? true
```

```suggestion
, dbusSupport ? true
, dbus
```
and then it should be removed above

---

same for your dbusSupport PRs btw

---

Does systemd work on anything other than Linux?  I didn't think so but I could be wrong.

There are seven other packages with `systemdSupport` defaulting to `stdenv.isLinux`, and only one which defaults to `true` is some kind of tray widget for KDE.

---

Also I *think* that this way Darwin users should be able to just `nix-env -iA sway` and run it using `WLR_BACKENDS=headless` (and RDP) or `WLR_BACKENDS=x11` (and XQuartz).  But I don't have a Mac so I can't try this.

---

i think the github syntax highlighter does that because the comma color in the nix syntax highlighting is similar to the removed color